### PR TITLE
obey 80 char limit, other style changes

### PR DESCRIPTION
**Pull request by kevyang**

### CODE REVIEW

File: src/time/cc_wheel.c [hunk 1]
```diff
@@ -272,8 +274,8 @@ _timing_wheel_insert(struct timing_wheel *tw, struct timeout_event *tev)
     INCR(timing_wheel_metrics, timing_wheel_insert);
     INCR(timing_wheel_metrics, timing_wheel_event);
 
-    log_verb("added timeout event %p into timing wheel %p: curr tick %zu, "
-            "scheduled offset %zu", tev, tw, tw->curr, tev->offset);
+    log_vverb("added timeout event %p into timing wheel %p: curr tick %zu, "
+              "scheduled offset %zu", tev, tw, tw->curr, tev->offset);
```
Comment (thinkingfish) [inline src/time/cc_wheel.c]: My opinion is to always use the 8-character indentation for arguments, second line and beyond. Aligning arguments  after the opening parenthesis creates all kinds of odd indentations that are not editor friendly, even they can be more visually pleasing. E.g. with the style in line 277/278, whenever the name of the function is changed, potentially every line needs to be re-aligned.

Comment (thinkingfish) [inline src/time/cc_wheel.c]: vi's default behavior is to used a fixed size, which can be specified per language (specified by shiftwidth or something like that).

Both CLion and sublime text have a toggle for this: e.g. http://www.sublimetext.com/docs/2/indentation.html

The current style guide is using 8-character, no mention of align to bracket.

Comment (thinkingfish) [inline src/time/cc_wheel.c]: So the practical concern here is- unless people have their editors lined up to do align-to-bracket, we will probably just have slight inconsistencies or back-and-forth tweaks depending on who was editing and what setting was used.

I think fixed-indentation is fine for one line continuation. When there are multiple lines, to achieve a visually neat block, I'd use an extra line.

```
call_a_function(
        argument 1, argument 2, argument 3, argument 4,
        argument 1, argument 2, argument 3, argument 4
);
```

File: include/cc_array.h [hunk 2]
```diff
@@ -31,9 +31,9 @@ extern "C" {
 #define NELEM_DELTA 16
 
 
-/*          name                type                default           description */
-#define ARRAY_OPTION(ACTION)                                                                              \
-    ACTION( array_nelem_delta,  OPTION_TYPE_UINT,   NELEM_DELTA,      "max nelem delta during expansion" )
+/*          name               type              default      description */
+#define ARRAY_OPTION(ACTION)                                                                      \
+    ACTION( array_nelem_delta, OPTION_TYPE_UINT, NELEM_DELTA, "max nelem delta during expansion" )
```
Comment (thinkingfish) [inline include/cc_array.h]: Please don't remove these spaces, the fields are all `K*ind + 1` aligned, `ind` being 4, which is the indentation width (== 4 in our case). This is to tab between the fields easily

File: include/channel/cc_pipe.h [hunk 3]
```diff
@@ -29,14 +29,14 @@ extern "C" {
...
 
-#define PIPE_POOLSIZE 0
+#define PIPE_POOLSIZE 0 /* unlimited */
 
-/*          name                type                default         description */
-#define PIPE_OPTION(ACTION) \
-    ACTION( pipe_poolsize,      OPTION_TYPE_UINT,   PIPE_POOLSIZE,  "pipe conn pool size" )
+/*          name            type              default        description */
+#define PIPE_OPTION(ACTION)                                                         \
+    ACTION( pipe_poolsize,  OPTION_TYPE_UINT, PIPE_POOLSIZE, "pipe conn pool size" )
```
Comment (thinkingfish) [inline include/channel/cc_pipe.h]: ditto

File: include/channel/cc_tcp.h [hunk 4]
```diff
@@ -41,10 +41,10 @@ extern "C" {
 #define TCP_BACKLOG  128
 #define TCP_POOLSIZE 0 /* unlimited */
 
-/*          name            type                default         description */
-#define TCP_OPTION(ACTION)                                                                \
-    ACTION( tcp_backlog,    OPTION_TYPE_UINT,   TCP_BACKLOG,    "tcp conn backlog limit" )\
-    ACTION( tcp_poolsize,   OPTION_TYPE_UINT,   TCP_POOLSIZE,   "tcp conn pool size"     )
+/*          name          type              default       description */
+#define TCP_OPTION(ACTION)                                                          \
```
Comment (thinkingfish) [inline include/channel/cc_tcp.h]: ditto

File: include/stream/cc_sockio.h [hunk 5]
```diff
@@ -54,9 +54,9 @@ extern "C" {
 
 #define BUFSOCK_POOLSIZE 0 /* unlimited */
 
-/*          name                type                default             description */
-#define SOCKIO_OPTION(ACTION)                                                             \
-    ACTION( buf_sock_poolsize,  OPTION_TYPE_UINT,   BUFSOCK_POOLSIZE,   "buf_sock limit" )
+/*          name               type              default           description */
+#define SOCKIO_OPTION(ACTION)                                                        \
+    ACTION( buf_sock_poolsize, OPTION_TYPE_UINT, BUFSOCK_POOLSIZE, "buf_sock limit" )
```
Comment (thinkingfish) [inline include/stream/cc_sockio.h]: ditto

**Comment by kevyang** [comment]
closing, this is really old and outdated.

**Pull request closed without merging**